exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,7 @@
 // Declare your function first
+function divide(x, y) {
+    return x / y;
+}
```

Comment:
You didn't have to account for this, but what would happen if `y = 0` was passed in and you end up with `x / 0`? try doing that in JS and see what happens. With knowledge of conditionals (like `if` statements) how would you prevent that from occurring?

Reply:
Thanks. I updated